plugins/chat/assets/javascripts/discourse/components/chat/thread/header-button.hbs

```diff
@@ -0,0 +1,15 @@
+<LinkTo
```

Comment:
We could maybe come up with a better name here.

Chat::Thread::ThreadsListButton ?